.github/workflows/release.yml

```diff
@@ -0,0 +1,22 @@
+name: Release
+on:
+  release:
+    types: [ 'published' ]
+
```

Comment:
as a thought -- in tiledbsoma we have a publish to test-pypi for things like manual runs, nightlies, etc -- enabling us to catch issues _before_ taggint the release

Reply:
Sure I can add that.